Assets/UXF/Scripts/Etc/Tracker.cs

```diff
@@ -115,4 +161,12 @@ public enum TrackerUpdateType
     {
         LateUpdate, FixedUpdate, Manual
     }
+
+    /// <summary>
+    /// The possible states a tracker can be in.
+    /// </summary>
+    public enum TrackerState
+    {
+        On, Off, Paused, Uninitialised
```

Comment:
I thing Recording, Paused, Stopped are better states. I don't think `Uninitialised` is needed and `Stopped` should be default